src/utils/borrowPools/mainnetPools.js

```diff
@@ -2072,4 +2072,57 @@ export default [
       abi: tokensAbi.MIM,
     },
   },
+  {
+    icon: useImage(`assets/images/tokens/Stargate-USDT.png`),
+    name: "Stargate USDT",
+    contractChain: 1,
+    id: 42,
+    stabilityFee: 0.25,
+    interest: 0,
+    ltv: 99,
+    borrowFee: 0,
+    isSwappersActive: true,
+    is0xSwap: true,
+    cauldronSettings: {
+      isDegenBox: true,
+      strategyLink: false,
+      isDepreciated: false,
+      acceptUseDefaultBalance: false,
+      healthMultiplier: 10,
+      hasAccountBorrowLimit: true,
+      hasWithdrawableLimit: true,
+      leverageMax: 15,
+      localBorrowAmountLimit: false,
+      isCollateralClaimable: false,
+      claimCrvReward: false,
+      isPrivate: true,
+      privatelyFor: [GNOSIS_SAFE_ADDRESS],
+    },
+    contract: {
+      name: "WhitelistedCauldronV4",
+      address: "0x1062eB452f8C7A94276437ec1F4aAca9b1495B72",
+      abi: poolsAbi.WhitelistedCauldronV4,
+    },
+    token: {
+      name: "Stargate USDT",
+      decimals: 6,
+      address: "0x38EA452219524Bb87e18dE1C24D3bB59510BD783",
+      abi: tokensAbi.StargateUSDT,
+    },
+    pairToken: {
+      name: "MIM",
+      icon: useImage(`assets/images/tokens/MIM.png`),
+      decimals: 18,
+      address: "0x99D8a9C45b2ecA8864373A26D1459e3Dff1e17F3",
+      abi: tokensAbi.MIM,
+    },
+    swapContractInfo: {
+      address: "0x88650a9A0F8894C0bc0d6AE833A64f7E34644935",
+      abi: ERC4626LevSwapper,
+    },
+    reverseSwapContractInfo: {
+      address: "0xa5564a2d1190a141CAC438c9fde686aC48a18A79",
+      abi: ERC4626LiqSwapper,
```

Comment:
ERC4626? Wrong ABI I guess

Reply:
as I understand they just reused from the caldrons above for us need only swap - and the interface there is the same